integration/testdata/custom-test/skaffold.yaml

```diff
@@ -14,3 +14,10 @@ deploy:
  kubectl:
    manifests:
    - pod.yaml
+profiles:
+  - name: custom
+    test:
+      - image: custom-test-example
+        custom:
+          - command: echo "foo" > runtest
+
```

Comment:
Is this necessary?  Can't we just look for the `test` file created above?

Reply:
Want to keep the file names distinct for debugging purposes in case of test failures.